realm/realm-library/src/syncIntegrationTest/java/io/realm/objectserver/utils/HttpUtils.java

```diff
@@ -38,6 +38,16 @@
     // will forward this query to the host, running the integration test server on 8888
     private final static String START_SERVER = "http://127.0.0.1:8888/start";
     private final static String STOP_SERVER = "http://127.0.0.1:8888/stop";
+    private static boolean inited = false;
+
+    // Start the ROS service only once since there are problems with reseting it. Also it makes testing faster.
```

Comment:
Can you elaborate on the problems resetting it? 
This pattern here seems a bit dangerous as it will leave the server running after the last integration test is done. We can perhaps live with that for now though.